assets/models/system/deformable_detr_twostage_refine_r50_16x2_50e_coco/spec.yaml

```diff
@@ -19,4 +19,4 @@ tags:
     apply_deepspeed: 'false'
     apply_ort: 'false'
   task: object-detection
-version: 4
+version: 5
```

Comment:
Version bump is not required. We have not released version 4 of these models yet.